doc-content/enterprise-only/pages/building-custom-dashboard-widgets-creating-process-heatmap-components-proc.adoc

```diff
@@ -14,7 +14,7 @@ You can create heatmap components for a specific process in {CENTRAL}.
 
 .Procedure
 
-. Create a new {KIE_SERVER} dataset using the following steps:
+. Create a new {KIE_SERVER} datas et using the following steps:
```

Comment:
```suggestion
. Create a new {KIE_SERVER} data set using the following steps:
```

Reply:
Barbora, please note that I prefer to make the updates in the source myself so that's what I've done. I made this change.